Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -93,8 +93,9 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
             var currentDate = DateTime.UtcNow;
             await _dbContext.TestIpAddresses.Where(x => x.TestId == testObject.Id).ToListAsync();
             // if Test is not paused and current date is not greater than EndDate and machine IP address is in the list of test ip addresses of  then it returns true and test link exist otherwise link does not exist
-           
-                return testObject != null && DateTime.Compare(currentDate, testObject.EndDate) < 0 && !testObject.IsPaused && testObject.TestIpAddress != null && testObject.TestIpAddress.Any(x => x.IpAddress == userIp) ? true : false;
+            if (testObject.TestIpAddress != null)
+                return testObject != null && DateTime.Compare(currentDate, testObject.EndDate) < 0 && !testObject.IsPaused && testObject.TestIpAddress.Any(x => x.IpAddress == userIp) ? true : false;
+            else return testObject != null && DateTime.Compare(currentDate, testObject.EndDate) < 0 && !testObject.IsPaused ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unnecessary Boolean literal(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1125)